### ISSUE TITLE
Why is preMajor=true treating minor updates as patch updates?

### ISSUE THREAD

**Issue by MikeDabrowski** [conventional-changelog/conventional-changelog-config-spec#76]
Im working on a project still in 0.y.z version. For bugfixes we want to bump patch and for new features or changes - minor. Im following conventional commits and using standard-verstion to cut a release. I noticed it always bumps patch version even if only feat commits have been made. I debugged the code and turns out `config.preMajor` is set to true. Before I overwrite it I want to know why such behaviour seems to be intended ?

In other words - why in 0.y.z any major or minor change is disregarded and only patch is incremented?

```
if (config.preMajor && level < 2) {
        level++
      }
      ...
VERSIONS[2] //patch
```
